react-components/stories/util/CopyTextArea.stories.js

```diff
@@ -1,23 +1,15 @@
 /**
  * @author psarando
  */
-import React, { Component } from 'react';
-
-import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
-
-import { getCyVerseTheme } from '../../src/lib';
-import CopyTextArea from '../../src/util/CopyTextArea';
+import React, {Component} from "react";
+import CopyTextArea from "../../src/util/CopyTextArea";
 
 class CopyTextAreaTest extends Component {
     render () {
-        let textToCopy =
-`The Dark Arts better be worried,
-oh boy!`;
+        let textToCopy = `The Dark Arts better be worried, oh boy!`;
```

Comment:
This string intentionally had a newline in the middle, to demo multi-line copy text areas.

Reply:
Oh. Ok. I will add the new line.